api/lib/database-client-selector.js

```
@@ -0,0 +1,90 @@
+  const runtime = detectRuntime();
+
+  console.log(`[Database] Detected runtime: ${runtime}`);
+
+  switch (runtime) {
+    case "edge":
+    case "vercel":
+    case "browser":
+      // Use web client for Edge Functions and browser environments
+      const { createClient: createWebClient } = await import(
+        "@libsql/client/web"
+      );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)